test/format/ops.c

```diff
@@ -1816,7 +1823,10 @@ row_truncate(TINFO *tinfo)
     /* The code assumes we're never truncating the entire object, assert that fact. */
     testutil_assert(tinfo->keyno != 0 || tinfo->last != 0);
 
-    trace_op(tinfo, "truncate %" PRIu64 "-%" PRIu64 " start", tinfo->keyno, tinfo->last);
+    trace_op(tinfo, "truncate %" PRIu64 " {%.*s} to %" PRIu64 " {%.*s}", tinfo->keyno,
```

Comment:
I wasn't sure why we print this trace here when it's also printed in the `trace_op`s call on line 1852. Is there some benefit to printing details before we've called `key_gen`, or could we delete this line?